themes/theme-b2b/src/components/NumberField.styles.ts

```diff
@@ -2,18 +2,18 @@ import { ThemeComponent, cva } from '@marigold/system';
 
 export const NumberField: ThemeComponent<'NumberField'> = {
   group: cva([
-    'border-border-light rounded-sm border border-solid',
-    'group-hover/field:border-border-hover',
-    'group-readonly/field:group-focus/field:border-border-light group-readonly/field:group-focus/field:shadow-none',
-    'group-disabled/field:bg-bg-disabled group-disabled/field:text-text-disabled group-disabled/field:cursor-not-allowed ',
```

Comment:
"text-text-disabled" is still being used in the label element. Intentional?
<img width="1158" alt="Captura de pantalla 2024-01-16 a la(s) 20 30 21" src="https://github.com/marigold-ui/marigold/assets/156671137/201e68b3-66a9-463a-a07e-fa49a918c900">

Reply:
The change looks right, nice work!

Reply:
You just changed "text-text-disabled" to "text-text-base-disabled", right? Just to make sure I'm checking the right thing.